### PR TITLE
test: Add tests for server and config

### PR DESCRIPTION
**Pull request by abbeyhrt**
Added tests for just the server and the config. 

#### Changelog

**New**

config_test.go
server_test.go

### CODE REVIEW

**Review by joshblack — CHANGES_REQUESTED** [review]
Want to add in a check for the DB config as well in `config_test`?

File: .env.template [hunk 1]
```diff
@@ -1,5 +0,0 @@
-export DB_NAME=keep_up_dev
```
Comment (joshblack) [inline .env.template]: Did you mean to remove these?

File: internal/config/config_test.go [hunk 2]
```diff
@@ -56,3 +58,70 @@ func TestConfigFromEnv(t *testing.T) {
...
+	rawQuery := v.Encode()
+
+	if cfg.Postgres.RawQuery != rawQuery {
+		t.Errorf(
+			"expected sslmode: %s, instead got: %s",
+			rawQuery,
+			cfg.Postgres.RawQuery,
+		)
+	}
+
```
Comment (joshblack) [inline internal/config/config_test.go]: Extra newline

**Review by joshblack — APPROVED** [review]

**Comment by joshblack** [comment]
Just some formatting notes and one about `env.template` and then should be good to go 👍

File: .env.template (hunk 1 above)
Comment (abbeyhrt) [reply .env.template]: for some reason, i thought you had said that the .env.template file needed to be changed to .env?

**Pull request merged**